### ISSUE TITLE
Account blocked — Identified violating security policy

### ISSUE THREAD

**Issue by rgcv** [Shippable/support#4134]
### See image below:

![image](https://user-images.githubusercontent.com/4052080/36363292-bdf9b114-1533-11e8-9a24-ff3690f59474.png)

### Transcript:

An error occured, please choose an action.

Your account has been blocked as it was identified to be violating our security policy.
Please contact support if you feel this is a mistake.

### Comments:

I do feel like this was a mistake. I was wondering why this happened, and hoping I could find an answer here? Not a major issue, meaning whatever priority is assigned to this particular problem is low, at least on my behalf, but I'm paradoxically eager to know what happened.

Thank you in advance!

**Comment by manishas** [comment]
@rgcv your account has been unblocked. We have an algorithm for blocking a small number of accounts due to previous problems with virtual currency miners. It does have some false positives. 

Sorry about that :(

**Comment by rgcv** [comment]
@manishas I see. Understandable, no problem. Thanks for the support! All good now :relaxed:

**Comment by AdelUnito** [comment]
@manishas @ambarish2012  I'm facing the same issue. can you please unblock my account

**Comment by a-murphy** [comment]
@AdelUnito I've unblocked your account.  You should be able to log in now.

**Comment by AdelUnito** [comment]
Thanks

**Comment by workademy-github** [comment]
@a-murphy I know this pretty old but I get redirected to github when I click on support. I am facing the same issue here. workademy-github is our root company account, could you unblock it? sorry if this is not the right channel

**Comment by a-murphy** [comment]
@workademy-github I've unblocked your account.  In future, you can open a new issue when an account is blocked to avoid notifying everyone who has commented on existing issues.  The same account won't be blocked more than once.

**Comment by workademy-github** [comment]
Thanks!

**Comment by SergeySmolyarchuk** [comment]
@a-murphy I'm facing the same issue. can you please unblock my account?
